airsonic-main/src/main/java/org/airsonic/player/service/PlaylistService.java

```diff
@@ -296,8 +295,11 @@ private void importPlaylistIfUpdated(File file, List<Playlist> allPlaylists) thr
         }
         InputStream in = new FileInputStream(file);
         try {
-            importPlaylist(User.USERNAME_ADMIN, FilenameUtils.getBaseName(fileName), fileName, in, existingPlaylist);
-            LOG.info("Auto-imported playlist " + file);
+            // FIXME - With the transition away from a hardcoded admin account to Admin Roles, there is no longer
```

Comment:
Should this be fixed before merging this PR?

Reply:
No one has suggested a good solution to this issue, and I don't want fixing this security issue to wait on one.  This side effect has only become a problem because @muff1nman wanted not to keep a "primary" admin user, so I've changed the PR to comply with that request.  @muff1nman which would you prefer, keeping this version with this side effect, or going back to the original version of this PR that kept a "primary" admin user?

Reply:
I'm probably going to work on the solution I proposed in https://github.com/airsonic/airsonic/issues/233#issuecomment-513936052 at some point which would resolve this playlist issue as well as many others, but that is almost certainly going to be a low priority.

Reply:
For now could you chose the first admin user found? Really I'd like to see the playlist owner field be nullable but that will probably take some work todo.

Reply:
I'd rather not put in a half-baked hack, since it seems unlikely that playlist importing is all that usable in its current form anyway(*).  I'd rather get the security regarding the hardcoded admin account in, and then later get some focus on improving/fixing the playlist functionality.

(*) Is there even any documentation on how the playlist importing stuff is supposed to work that would help anyone actually use it?  For instance, how are files in the playlist supposed to have their paths specified?  How does it pick which media folder is used as the base? etc...

Reply:
I used it a year ago. I don't think the negligible security gained here is worth losing the feature over.

Reply:
@muff1nman So then would you prefer that I revert back to the original version of this PR that kept a "primary" admin user, in order to preserve the playlist functionality?  I don't want to put playlist overhaul in as part of this PR.

Reply:
Nope I'd prefer you kept it as is. For the playlist user you can:

1. Use the first admin user
2. Allow the playlist import user to be configurable
3. Change the playlist to allow a nullable user field

Although I still think 3 would be ideal/workable, 1 or 2 would be considerably easier and be much better than straight out removing this functionality.

Reply:
As requested, PR has been modified to use the first admin account for playlist imports.